napari/_qt/dialogs/screenshot_dialog.py

```diff
@@ -27,9 +27,11 @@ def __init__(
         self,
         save_function: Callable[[str], Any],
         parent=None,
-        directory=str(Path.home()),
```

Comment:
`str(Path.home())` is immutable, so this change seems not needed, and even a regression as now you recompute the default value at each call of the function.

If you don't want a call here, you can make it a global variable.

Reply:
changed